### DISCUSSION TITLE
Achievements badges

### DISCUSSION THREAD

**Discussion by kolumnin** [community/community#28827] [Other Feature Feedback, Questions, & Ideas]
What are the applications of the Achievements Badges?

**Comment by radar-blip** [comment]
Like, what do you mean?

**Comment by ms-Almazid** [comment]
MESHAL

**Comment by lifeparticle** [comment]
Visualization of your GitHub activity/achievements.

**Comment by melvincwng** [comment]
@kolumnin, they are mostly for cosmetic purposes.

If you need more information on achievements, you can visit the link [here](https://github.com/Schweinepriester/github-profile-achievements)

Hope this helps.
